### PR TITLE
Scrollable list of pinned messages

### LINKED ISSUE #15380: The message with long text is not scrollable in the list of pinned messages
#### Steps to reproduce:
1. Send a few messages with a long text in any chat
2. Pin messages
3. Open a list of pinned messages
4. Scroll it

#### Actual result:
The pinned list is not scrollable on the androids

For IOS:

https://user-images.githubusercontent.com/52490791/225582290-2c2b3d7e-c6cc-48f8-9c5f-d4c460b0cbac.mp4

For Android:

https://user-images.githubusercontent.com/52490791/225582176-0654398d-9574-41d2-950a-c93e54206649.mp4

#### Expected result:
Long data is scrollable on the pinned messages list

### PR DESCRIPTION
**Pull request by alwx**
Fixes #15380

Later a new component for pinned messages will be designed (we discussed it here: https://github.com/status-im/status-mobile/issues/14955), there is even an issue for that created already and I already did some work for it: https://github.com/status-im/status-mobile/issues/14564

This solution will allow us to quickly unblock us and make sure pinned messages component works as expected (+ it's also a good start for a new component)

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/menus/pinned_messages/style.cljs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   :align-items     :center
+   :margin-top      20
+   :margin-bottom   bottom-inset})
+
+(def no-pinned-messages-icon
+  {:width           120
+   :height          120
+   :justify-content :center
+   :align-items     :center
+   :border-width    1})
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/menus/pinned_messages/style.cljs:41]: What's the border color ?

File: src/status_im2/contexts/chat/menus/pinned_messages/view.cljs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            [status-im2.contexts.chat.messages.content.view :as message]
+            [status-im2.contexts.chat.menus.pinned-messages.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(def list-key-fn #(or (:message-id %) (:value %)))
+
+(defn message-render-fn
+  [{:keys [deleted? deleted-for-me?] :as message} _ _ context]
+  ;; TODO (flexsurfer) probably we don't want reactions here
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/menus/pinned_messages/view.cljs:16]: Do we need this comment ?

File: src/status_im2/contexts/chat/menus/pinned_messages/view.cljs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  (if (or deleted? deleted-for-me?)
+    [content.deleted/deleted-message message context]
+    [message/message-with-reactions message context]))
+
+(defn pinned-messages
+  [chat-id]
+  (let [pinned-messages        (rf/sub [:chats/pinned-sorted-list chat-id])
+        render-data            (rf/sub [:chats/current-chat-message-list-view-context :in-pinned-view])
+        current-chat           (rf/sub [:chat-by-id chat-id])
+        {:keys [community-id]} current-chat
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/menus/pinned_messages/view.cljs:26]: You can do the destructuring on line 25

**Review by ibrkhalil — APPROVED** [review]
Non blocking comments

File: src/status_im2/contexts/chat/menus/pinned_messages/view.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im2/contexts/chat/menus/pinned_messages/view.cljs:16]: yes, we don't want reactions

**Comment by pavloburykh** [comment]
@alwx thanx for the PR. Tested and ready to be merged. 

I have pushed a small [commit](https://github.com/status-im/status-mobile/pull/15506/commits/800fb647fdf5a88012503e0b1c832871c6a5a0f5) with e2e fix into your PR because accessibility id for pinned list has changed which resulted into e2e failures. Could you please include my commit during merge? Thank you!

**Pull request merged**